frank/utilities.py

```diff
@@ -729,3 +729,63 @@ def convolve_profile(r, I, disc_i, disc_pa, clean_beam,
     I_smooth /= counts
 
     return I_smooth
+
+
+def generic_dht(x, f, Rmax=2.0, N=1000, direction='forward', inc=None):
```

Comment:
This should also be able to return the transform at specified points (not just the collocation points).